app/components/Header.tsx

```diff
@@ -15,7 +14,7 @@ export const Header = () => {
   return (
     <header className="flex justify-between items-center max-w-full w-full py-8">
       <Link className="home text-3xl font-medium no-underline flex-1" to="/">
-        {siteMetadata.domain}
+        <img alt="Project logo" src="https://user-images.githubusercontent.com/89982193/189909779-fc246b9e-1ff8-46d8-9c57-8c51d1676d77.png" />
```

Comment:
Instead of this, I think a good idea would be adding the logo URL to the `siteMetadata` file to keep all the metadata together in one place.

Also, what about displaying the domain as a fallback if there is no logo?

Reply:
That's great. I will make those changes.

Does `siteMetadata` contain logo URL ?

Reply:
No, but you can add a new property, something like:

`logo: "https://user-images.githubusercontent.com/89982193/189909779-fc246b9e-1ff8-46d8-9c57-8c51d1676d77.png"`